### ISSUE TITLE
Saving regex should save examples and substitutions

### ISSUE THREAD

**Issue by Jakobud** [firasdib/Regex101#205]
Right now, when you save an regex to your account, it only saved the regex code itself. It should really save EVERYTHING. Your regex, your examples, your substitutions, etc... That would be REALLY nice. I hate coming back to a regex to make adjustments and have to input all my examples again to make sure I don't create regression bugs.

**Comment by firasdib** [comment]
Thats odd... Everything should be saved. 

You're trying to tell me that when you sign in and save a regex, only the regex and nothing else is saved when you revisit that link?

**Comment by Jakobud** [comment]
Oh you know what, it was all saved actually. I didn't realize this before. The problem is that your UI doesn't make it clear that it was saved. I thought when I saved a regex I got just the following screen and thats it:

![image](https://cloud.githubusercontent.com/assets/87356/5632190/0778f5ca-958a-11e4-97bf-345e0c6e1a68.png)

Now I see when I click `FW0DU7/1` it takes me to the regex. I had no idea that was a clickable link honestly. Maybe make that more clear.

I love this website. Such a fantastic tool. Best regex site online for testing and learning, hands down. Keep up the good work!
